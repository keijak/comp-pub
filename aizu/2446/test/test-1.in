20 1000000000000000000
1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
23 84 11 49 70 2 58 71 20 10 17 57 98 9 32 39 10 21 45 41
