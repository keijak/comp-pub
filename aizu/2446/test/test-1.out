999999103729729034.234874785905930240000000000000000000
